helm_deploy/elite2-api/templates/_envs.tpl

```diff
@@ -9,90 +9,90 @@ env:
       secretKeyRef:
         key: APPLICATION_INSIGHTS_IKEY
         name: {{ template "app.name" . }}
-  - name: DATA_COMPLIANCE_INBOUND_DELETION_SQS_AWS_ACCESS_KEY_ID
+  - name: DATA_COMPLIANCE_REQUEST_SQS_AWS_ACCESS_KEY_ID
```

Comment:
Will need to update secrets in AWS Secrets Manager?

Reply:
It is done :)

Reply:
I have now, thank you!